src/cljs/rems/administration/catalogue_items.cljs

```diff
@@ -112,9 +112,11 @@
            :form (let [value (:form-name item)]
                    {:value value
                     :td [:td.form
-                         [atoms/link nil
-                          (str "/administration/forms/" (:formid item))
-                          value]]})
+                         (if value
+                           [atoms/link nil
+                            (str "/administration/forms/" (:formid item))
+                            value]
+                           [text :t.administration/no-form])]})
```

Comment:
The to-catalogue-item helper was for a "View" button like we have on the catalogue item list page. A helper for styled links like this might be nice, I agree. However this case is a bit special because it doesn't use `:form/id` but the `:formid` from the catalogue item api.

Reply:
URN/EGA link stuff does the iffing inside so both formats are supported with the same "API". I think that's feasible to make the API nice to use. However since we don't do this for links yet it's ok to not do it.